app/Helpers/Geocoder.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Helpers;
+
+use Illuminate\Support\Facades\Log;
+
+/**
+ * Google Geocoder interface.
+ */
+class Geocoder
+{
+
+    protected $apikey = "AIzaSyDb1_XdeHbwLg-5Rr3EOHgutZfqaRp8THE";
```

Comment:
This would be better coming from the .env file (although it wasn't previously it turns out!)

Reply:
Agreed, just not entirely sure how to do this with Laravel, will look into it.